From ed1dd57ff45a059bc3f54c96ac9e41bbc002186f Mon Sep 17 00:00:00 2001
From: Vladimir 'phcoder' Serbinenko <phcoder@gmail.com>
Date: Sun, 10 Mar 2013 18:49:05 +0100
Subject: [PATCH 203/364] 	* grub-core/video/i386/pc/vbe.c 
 (grub_video_vbe_print_adapter_specific_info): Replace division by 
 shifts.

---
 ChangeLog                     | 6 ++++++
 grub-core/video/i386/pc/vbe.c | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 51bc363..48ca30e 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,11 @@
 2013-03-10  Vladimir Serbinenko  <phcoder@gmail.com>
 
+	* grub-core/video/i386/pc/vbe.c
+	(grub_video_vbe_print_adapter_specific_info): Replace division by
+	shifts.
+
+2013-03-10  Vladimir Serbinenko  <phcoder@gmail.com>
+
 	Adjust types in gdb module to have intended unsigned shifts rather than
 	signed divisions.
 
diff --git a/grub-core/video/i386/pc/vbe.c b/grub-core/video/i386/pc/vbe.c
index e8a8c7a..f112f15 100644
--- a/grub-core/video/i386/pc/vbe.c
+++ b/grub-core/video/i386/pc/vbe.c
@@ -1197,7 +1197,7 @@ grub_video_vbe_print_adapter_specific_info (void)
   
   /* The total_memory field is in 64 KiB units.  */
   grub_printf_ (N_("              total memory: %d KiB\n"),
-		(controller_info.total_memory << 16) / 1024);
+		(controller_info.total_memory << 6));
 }
 
 static struct grub_video_adapter grub_video_vbe_adapter =
-- 
1.8.1.4

